Jenkinsfile

```diff
@@ -34,7 +34,7 @@ for (lbl in labels) {
                 condaEnv("isis3") {
                     // Environment
                     loginShell """
-                        conda install -c conda-forge python=3
+                        conda install -c conda-forge python=3 findutils
```

Comment:
temporary fix, PR into the docker repo incoming. Not sure how that process will go to get the new containers running so I don't want to count on it too much.